lib/tests/streamlit/error_util_test.py

```diff
@@ -17,35 +17,12 @@
 import contextlib
 import io
 import unittest
-from unittest.mock import patch
 
 from streamlit.error_util import _print_rich_exception, handle_uncaught_app_exception
-from streamlit.errors import UncaughtAppException
 from tests import testutil
 
 
 class ErrorUtilTest(unittest.TestCase):
-    @patch("streamlit.error_util._show_exception")
-    def test_uncaught_exception_show_details(self, mock_show_exception):
-        """If client.showErrorDetails is true, uncaught app errors print
-        to the frontend."""
-        with testutil.patch_config_options({"client.showErrorDetails": True}):
```

Comment:
It looks like there is an equivalent test for `False` added [here](https://github.com/streamlit/streamlit/pull/9909/files#diff-5b083edb9725793b64aa5ae9f1eac9860d360633979b147e05bb6e675300f649R168), but is `True` also still covered in a test?